src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks.
You won`t be able to reuse block in a proper way in such case.